components/TradingCardGrid.jsx

```diff
@@ -45,9 +93,15 @@ export default function TradingCardGrid({ teamMembers }) {
           numResults={filteredTeamMembers.length}
           numTotal={teamMembers.length}
         />
-        <ViewToggle />
+        <ViewToggle
+          currentView={view}
+          onChangeView={handleChangeView}
+        />
       </div>
-      <div className="cmp-trading-card-grid__grid">
+      <div className={`cmp-trading-card-grid__grid ${gridClassModifier}`}>
+        <Show when={!filteredTeamMembers.length}>
+          <p className="cmp-trading-card-grid__status">No team members fit your criteria.</p>
```

Comment:
This line is redundant now that we have the `TradingCardNumResults` component. "0 your X meet your criteria" is plenty to let people know they no one matches